source/options/GameplaySettingsSubState.hx

```diff
@@ -62,6 +62,13 @@ class GameplaySettingsSubState extends BaseOptionsMenu
 			'bool',
 			true);
 		addOption(option);
+		
+		var option:Option = new Option('Instant Respawn',
+			"If checked, you will be respawned automatically, without animation.",
```

Comment:
Potentially change to: "If checked, you will automatically respawn, skipping the game over animation"?

Reply:
OK, that will give more information, what this option does.